nservicebus/gateway/scale-out.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Gateway And Scale-Out
+summary: Explanation of how the Gateway handles Scale-Out
+tags: []
+related:
+ - samples/gateway
+---
+
+The gateway is designed to run on a single instance for each logical endpoint. A scaled out endpoint needs one instance to receive all the incoming HTTP gateway traffic for that endpoint. There are different strategies to achieve this based on the transport and the scale-out mechanism utilized:
+
+### MSMQ with the distributor
+
+With MSMQ scale out using the [distributor](/nservicebus/scalability-and-ha/distributor), the gateway runs on the master node/distributor, making it the natural single receiving gateway instance for the endpoint. Running the master node/distributor as [a clustered service on a Windows Failover Cluster](/nservicebus/scalability-and-ha/deploying-to-a-windows-failover-cluster.md) provides High Availability(HA). No special action must be taken other than configuring senders to send gateway HTTP traffic to the master node instead of directly to the workers.
+
+### Others - Brokered transports and MSMQ scaled out with new unified scalability model in version 6
+
+With [brokered transports](/nservicebus/scalability-and-ha/scale-out.md#versions-5-and-below-sql-server-and-rabbitmq) or the [new unified scalability model](/nservicebus/scalability-and-ha/sscale-out.md#versions-6-and-above) in version 6, there is no natural single receiving gateway instance for each logical endpoint. One needs to be explicitly selected and any endpoints sending to a gateway must be configured to use the HTTP address of the selected instance.
+
+Place a HTTP Load Balancer(LB) in front to avoid hard coding individual endpoint instance gateway HTTP addresses in gateway senders. Configure the LB to forward all traffic to the HTTP address of the selected endpoint instance and sending endpoints to send through the LB. 
```

Comment:
> in front

of what? The usage of this expression I found were "behind/in front" of firewall. How about:

"To avoid hard-coding (endpoint instance?) addresses in senders, use HTTP Load Balancer (LB). Configure LB to forward all traffic to the selected endpoint instance. Then configure all senders (or sending endpoints?) to send messages to LB.